demo/App.tsx

```diff
@@ -51,53 +61,91 @@ const App = () => {
             <View
               style={{
                 width: '45%',
-                backgroundColor: 'skyblue',
+                marginBottom: 15
+              }}>
+              <Button
+                color="red"
+                testID="runNativeError"
+                accessibilityLabel="runNativeError"
+                onPress={() => RaygunDemoBridge.runNativeError()}
+                title="Run Native Error"
+              />
+            </View>
+
+            <View
+              style={{
+                width: '45%',
+                backgroundColor: 'orange',
                 marginBottom: 15
               }}>
               <Button
                 testID="triggerUndefinedErrorBtn"
                 accessibilityLabel="triggerUndefinedErrorBtn"
                 onPress={() => {
-                  //@ts-ignore
-                  global.undefinedFn();
+                  throw new Error("Test Error: Uncaught error");
                 }}
-                title="Trigger undefined error"
+                title="Trigger Uncaught Error"
               />
             </View>
+              <View
+                  style={{
+                      width: '45%',
+                      backgroundColor: 'yellow',
+                      marginBottom: 15
+                  }}>
+                  <Button
+                      testID="triggerUndefinedErrorBtn"
+                      accessibilityLabel="triggerUndefinedErrorBtn"
+                      onPress={() => {
+                          //@ts-ignore
+                          RaygunClient.sendRUMTimingEvent(RealUserMonitoringTimings.ViewLoaded, "TestEvent", 100)
+                      }}
+                      title="SEND CUSTOM RUM TIMING EVENT"
+                  />
+              </View>
             <View
               style={{
                 width: '45%',
-                backgroundColor: 'skyblue',
+                backgroundColor: 'yellow',
                 marginBottom: 15
               }}>
               <Button
                 testID="triggerCustomErrorBtn"
                 accessibilityLabel="triggerCustomErrorBtn"
                 onPress={() => {
-                  throw new Error("CUSTOM MESSAGE!");
+                  // Current user of the application
+                  const user = "Guest"
+
+                  // You caught some error, and now you can send it away
+                  const customData: CustomData = {"Who's to blame: ": user}
```

Comment:
IMO: It might be nice to provide a different example of using custom data, as users can already implement user tracking separately. Also I would avoid using the terminology "who's to blame" (not very descriptive and slightly less professional).